app/models/user.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = (sequelize, DataTypes) =>
+  sequelize.define(
+    'User',
+    {
+      firstName: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        field: 'first-name'
+      },
+      lastName: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        field: 'last-name'
```

Comment:
This is spinal-case, are you sure this work ?

Reply:
No! I actually corrected it on the other branch were I actually tested the user creation.